### ISSUE TITLE
Australia's first unofficial_name is now "Australien"?

### ISSUE THREAD

**Issue by jonbuda** [countries/countries#765]
a customer of ours just noticed this since we use the `unofficial_names` list because the names are shorter, but it looks like the first unofficial_name in the list of Australia is now "Australien"? Perhaps this wasn't intentional for the default non-translated AU.yml?

https://github.com/countries/countries/commit/f28e2e1c5604ec5a6305db87dc892765abc6229d

**Comment by evolve2k** [comment]
This would seem to be correct, Australien is used in German, France and Sweden as the localised word for Australia.
It would seem you might want to be using the Common Name.

From the README, [names & translations section](https://github.com/countries/countries#names--translations)

```ruby
c.iso_long_name # => "The United States of America"
c.iso_short_name # => "United States of America"
c.common_name # => "United States" (This is a shortcut for c.translations('en'))
c.unofficial_names # => ["United States of America", "Vereinigte Staaten von Amerika", "États-Unis", "Estados Unidos"]
```

**Comment by pmor** [comment]
Yes, the `unofficial_names` attribute has no set order or structure, the main purpose of it is to help with search methods such as `ISO3166::Country.find_country_by_any_name` where you might be using user input that doesn't match any of the official or common name attributes.

Generally speaking, `common_name` is probably going to have the expected output, as some countries have a rather long `iso_short_name`
